docs/ff-integrations/ads/admob.md

```diff
@@ -101,43 +84,50 @@ To add the APP keys:
+- **Ad Content Filtering**: To filter the type of ads displayed, select the appropriate content rating. AdMob will ensure that ads returned for these requests have a content rating at or below the level selected.
+    
+    These are the levels you can set:
+    
+    - **G (General Audience)**: Suitable for all audiences, with no adult content or explicit themes.
+    - **PG (Parental Guidance)**: Ads may contain mild content, suitable for children with parental supervision.
+    - **T (Teen)**: Ads with content appropriate for teenagers; may include some mature topics.
+    - **MA (Mature Audience)**: Ads intended for adults, which may include strong themes or explicit content.
 
-Showing AdBanner From AdMob comprises of following steps:
+Once the setup is completed, you can start to display [AdBanner](#adbanner) or [Interstitial ads](#interstitial-ad) in your app.
 
-1. [Adding AdBanner widget](#1-adding-adbanner-widget)
-2. [Assigning Ad Unit ID](#2-assigning-ad-unit-id)
-3. [Enabling test ads](#3-enabling-test-ads)
-4. [Testing ads](#4-testing-ads)
+## AdBanner
 
-### 1. Adding AdBanner widget
+The **AdBanner** widget displays advertisement banners within your app. It can feature text, images, and rich media, including video ads.
 
-To add the AdBanner widget to your project:
+To display an **AdBanner** from AdMob, follow these steps:
 
-1. Add the Column widget from the widget tree or from the canvas area.
-2. Drag the **AdBanner** widget from the **Base Elements** tab (in the Widget Panel) or add it directly from the widget tree.
-3. Move to the property editor and scroll down to the **Ad Banner Dimensions** section and adjust the **Width** and **Height** property.
+### Adding AdBanner widget
+
+First, add the **AdBanner** widget from the **Base Elements**. Next, create a new Banner Ad unit in AdMob, then copy and paste its **unit ID** into FlutterFlow. The Ad unit ID is a unique identifier assigned to each ad created in AdMob.
+
+:::tip
+
+While building your app, clicking on too many ads may cause your AdMob account to be flagged for invalid activity. To avoid this, it's recommended to enable **Test Ads** during development.
 
-:::info
-If you leave the Width property empty, AdBanner will take the full width of the screen.
 :::
+
 <div style={{
     position: 'relative',
     paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
     height: 0,
     width: '100%'}}>
     <iframe 
-        src="https://demo.arcade.software/PBkC8jgpGHT2zrPhl5yh?embed&show_copy_link=true"
+        src="https://demo.arcade.software/vlm11muoQuU7YRS2AzQW?embed&show_copy_link=true"
```

Comment:
In this Arcade, there is an info about ad banners having a default dimension of X. I think that info should exist out of Arcade as an admonition .

Reply:
ok, added into info admonition.